apps/public-docsite-v9/src/Concepts/AdvancedStylingTechniques.stories.mdx

```diff
@@ -49,10 +49,12 @@ export function App() {
 }
 ```
 
-A little scaffodling is required to get here first. Define a `useFancyButtonStyles.ts`, and calculate the style similar to how you would for any other Fluent component:
+A little scaffolding is required to get here first. Define a `useFancyButtonStyles.ts`, and calculate the style similar to how you would for any other Fluent component.
+
+> ⚠️ Custom style hooks must also append the slot's original className prop as returned by `getSlotClassNameProp_unstable`, after the custom styles. This ensures that the className prop added by the user will take precedence over custom styles.
```

Comment:
Following up from https://github.com/microsoft/fluentui/pull/34166/files#r2044847081
cc @ling1726 @micahgodbolt @mltejera 

Any further opinions about whether this should be included in the public docsite? For reference, here's my last relevant comment in the linked thread:
> I wouldn't necessarily call this advanced usage. At least, it's not more advanced than any of the rest of the custom styling hook API. In fact, it's (currently) required to use this function to correctly implement a custom style hook.

Reply:
Ping @ling1726 @micahgodbolt @mltejera, I'd like to resolve this if possible. Should we add this to the public documentation? IMO we should since a custom style hook is technically not correct without using it. Thanks!